src/Microsoft.FeatureManagement/Microsoft.FeatureManagement.csproj

```diff
@@ -37,6 +37,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="2.1.23" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="2.1.10" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.1.1" />
```

Comment:
Why is it needed?

Reply:
ActivatorUtilities?

Reply:
Not used. Thought it was needed at some point. Removing now